graphql/appointment/mutations.ts

```diff
@@ -51,4 +66,23 @@ export class MutateAppointmentResolver {
         await createGroupAppointments(subcourseId, appointments);
         return true;
     }
+
+    @Mutation(() => Boolean)
+    @AuthorizedDeferred(Role.OWNER)
+    async appointmentUpdate(@Ctx() context: GraphQLContext, @Arg('appointmentToBeUpdated') appointmentToBeUpdated: AppointmentUpdateInput) {
+        const appointment = await getLecture(appointmentToBeUpdated.id);
+        await hasAccess(context, 'Lecture', appointment);
+        const currentDate = moment();
+        const isPastAppointment = moment(appointment.start).isBefore(currentDate);
```

Comment:
Add duration? Or should one not edit an appointment while it is ongoing?

Reply:
Good point 👍🏻 and valid case